documentation/ssl_server_certificate_change_and_virtual_ips.markdown

```diff
@@ -3,7 +3,7 @@ layout: default
 title: "Puppet Server: Known Issues: SSL Server Certificate Change and Virtual IP Addresses"
 canonical: "/puppetserver/latest/ssl_server_certificate_change_and_virtual_ips.html"
 ---
-[pe_db_instructions]: https://docs.puppetlabs.com/pe/latest/release_notes_known_issues.html#puppetdb-behind-a-load-balancer-causes-puppet-server-errors
+[pe_db_instructions]: https://docs.puppetlabs.com/pe/latest/release_notes_known_issues.html#puppetdb-behind-a-load-balancer-causes-puppetserver-errors
```

Comment:
Looks like this is a dead link anyway.  I don't see this page in the Known Issues for any PE releases past PE 3.8, for some reason.  Maybe this should be linked instead to https://docs.puppet.com/pe/3.8/release_notes_known_issues.html#puppetdb-behind-a-load-balancer-causes-puppet-server-errors for now?  @gguillotte - what do you think?